request-processor/src/application/core/pipeline.py

```
@@ -109,6 +110,7 @@ def fetch_response_data(
             )
         except Exception as err:
             logger.error("An exception occured during Pipeline Transform: ", str(err))
```

Ruff (0.15.13)

[error] Too many arguments for `logging` format string

(PLE1205)